obal/data/roles/setup_sources/tasks/git.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: 'Set source checkout location'
+  set_fact:
+    setup_sources_git_checkout_dir: "/tmp/ansible_git_{{ inventory_hostname }}"
+
+- name: 'Ensure git location'
+  file:
```

Comment:
Does the git module not create the directory by default?

Reply:
yup, this was a relic, removed